spec/factories/state_file_ny_intakes.rb

```diff
@@ -116,6 +116,7 @@
 
     raw_direct_file_data { StateFile::DirectFileApiResponseSampleService.new.old_xml_sample }
     raw_direct_file_intake_data { StateFile::DirectFileApiResponseSampleService.new.old_json_sample }
+    df_data_import_succeeded_at { DateTime.now }
```

Comment:
oh maybe i should've added this when i added the column, it makes sense -- except in situations we may want an unimported intake for testing? but then I guess we manually set the `df_data_import_succeeded_at` to `nil`

Reply:
Agreed, the default should be success and we can override with nil on this and the raw_*_data columns when testing failure